### ISSUE TITLE
Last Commentary-related Nav Issues

### ISSUE THREAD

**Issue by Tsuey** [Tsuey/L4D2-Community-Update#255]
Resolving the last of stuff from PR #131 items that needed review:

# Commentary blockers we'll delete after nav is fixed

### Dead Center 4

`_commentary_infopanel`: Blocks standing on the info panel as godspot. Note that it is destroyed for one of the outros. Seems to be the most commonly complained about Commentary clip.

![image](https://user-images.githubusercontent.com/71397129/168412056-dc6e9c27-b472-41c9-a8ec-0427cc684fa9.png)

### Dark Carnival 1

`_commentary_slopetruck`: Blocks standing on the truck at the top of the slope by the saferoom, godspot.

![image](https://user-images.githubusercontent.com/71397129/168412088-369d8fe4-11b4-4774-8207-a6e47a3483a0.png)

### Dark Carnival 3

Implement and obsolete [this VScript](https://github.com/Derpduck/L4D2-Comp-Stripper-Rework/blob/master/scripts/vscripts/nav_fixes/c2m3_coaster_navfixes.nut) into the navmesh at the Coaster fences to improve Common nav. Technically this clip is `_commentary_fencewalk_a` but there'd only be a reason to delete it for Scavenge.

![image](https://user-images.githubusercontent.com/71397129/168412349-62da550f-9aa7-4f6f-bb9d-2ad0adae19c3.png)

### Parish 3

`_commentary_nav_parkingtruck_*`: Blocks standing on 2 of the trucks in the parking lot, clearly a case of Commentary patching without bothering to update the nav file.

These have no or few nav connections:

![image](https://user-images.githubusercontent.com/71397129/168412420-c12e7e3a-0887-48e3-8400-954f288e57ac.png)

Since it seems *a little unusual* that we'd be unblocking the above trucks, there is precedent for it since this truck further back was never blocked -- it could use some nav improvement too, though:

![image](https://user-images.githubusercontent.com/71397129/168412469-a7f35c93-e79b-475e-86d9-97ee77b05042.png)

# Ignore / already fixed / just FYI

Hard Rain 2's ``_commentary_spool`` nav connections are very strong, but I noticed Hard Rain 3's spool doesn't have any nav:

![image](https://user-images.githubusercontent.com/71397129/168412639-87175fc5-9743-4881-bf98-28dd43ae8b1f.png)

Similar with the washer box table: Hard Rain 1 table has no nav, but Hard Rain 5 does.

I suspect that's intentional, just FYI.

**Comment by alexiscoutinho** [comment]
Would bsp clips also be removed if their purpose was to block a God spot? Are there any of this type though?

**Comment by Tsuey** [comment]
> Would bsp clips also be removed if their purpose was to block a God spot? Are there any of this type though?

So far (with Commentary blockers) everything we're opening up was already open at some point in the past. That might be a question we don't know how to answer yet.

A small amount of Commentary work was already done with TLS, like unblocking the c2m3 swan room shelf and instead fixing the nav.
